amundsen_application/static/js/components/NavBar/styles.scss

```diff
@@ -1,5 +1,7 @@
 @import 'variables';
 
+$navbar-item-line-height: 36px;
```

Comment:
Thanks Marcos! Mind adding this variable to https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/css/_variables-default.scss#L89-L93?

Reply:
Sure!